driver/modern_bpf/helpers/extract/extract_from_kernel.h

```diff
@@ -769,7 +769,6 @@ static __always_inline bool extract__exe_from_memfd(struct file *file)
 		return false;
 	}
 
-#pragma unroll
```

Comment:
@Andreagit97 why remove it? Seems unrelated to the kmod fix and why only do in modern_bpf, it's typically used btw.

Reply:
in the modern bpf, we don't use the unroll, it is something we have to use in the old one because bounded loops were introduced in kernel 5.3 :/ IIRC there are no other places in the modern probe in which we use the unroll :thinking: